include/swift/AST/DiagnosticsSema.def

```diff
@@ -3680,10 +3680,10 @@ ERROR(dynamic_self_default_arg,none,
 
 ERROR(attr_only_one_decl_kind,none,
       "%0 may only be used on '%1' declarations", (DeclAttribute,StringRef))
+ERROR(attr_must_be_on_valid_parameters,none,"'%0' may only be used on %1 parameters", (StringRef, StringRef))
```

Comment:
Please format it the same way as the reset of the declarations in this file and a better name could be `attr_only_valid_on_func_or_init_params`.

Reply:
Thanks!